nextflow_schema.json

```diff
@@ -167,6 +167,17 @@
                 }
             }
         },
+        "new_group_1": {
+            "title": "New Group 1",
```

Comment:
```suggestion
            "title": "AMP-AMPlify-Predict",
```

Reply:
Where did this "New Group 1" came from? Somehow these changes in the nextflow_schema.json made the checks fail ...

Reply:
I think the group name didn't get saved when you changed it in the builder